languages/js/src/typeorm_adapter.ts

```diff
@@ -0,0 +1,94 @@
+import { Connection, SelectQueryBuilder } from 'typeorm';
```

Comment:
nit: file should be named `typeOrmAdapter.ts` to match the primary export

Reply:
Also the `typeorm` dependency listed in `devDependencies` in `languages/js/package.json` won't be installed when someone installs the published `oso` package. I think there'll be an error about `typeorm` missing when anyone imports from `oso` and doesn't already have TypeORM installed in their project, regardless of whether the project actually uses TypeORM. Definitely need to test this prior to shipping.

Reply:
Ok then I'm just going to keep the adapter in the tests for now and people can copy it if they want to

Reply:
Can we avoid this somehow? In python you can conditionally import and handle failure gracefully. Can we do that here?

Reply:
Avoided by removing the top-level export. Anyone wishing to can still `import { typeOrmAdapter } from 'oso/dist/src/typeOrmAdapter'`. It's ugly but good enough for the time being.